src/renderer/components/screens/AccountListSidebar/styles.module.scss

```diff
@@ -0,0 +1,203 @@
+@import '../../../../../scss/variables';
+
+.avatar {
+  --local-avatar-size: 48px;
+  --local-avatar-vertical-margin: 8px;
+  --local-avatar-font-size: 26px;
+
+  position: relative;
+  user-select: none;
+  display: inline-block;
+  font-weight: normal;
+
+  height: var(--local-avatar-size);
+  width: var(--local-avatar-size);
+  margin: var(--local-avatar-vertical-margin) 0;
+
+  div.content,
+  img.content {
+    height: var(--local-avatar-size);
+    width: var(--local-avatar-size);
+    border-radius: 100%;
+    pointer-events: none;
+  }
+
+  div.content {
+    background-color: #505050;
+    line-height: var(--local-avatar-size);
+    object-fit: cover;
+    color: var(--avatarLabelColor);
+    text-align: center;
+    font-size: var(--local-avatar-font-size);
+  }
+
+  img.content {
+    object-fit: cover;
+  }
+}
```

Comment:
`.avatar` is not used by `AccountListSidebar` and seems to be a duplicate as well

Reply:
it is used by `AccountItem` and all sidebar components share this style file. the class on the top of the file is copied from the other avatars of delta chat desktop, cause the new styling system does not allow me to overwrite classes from different files.